src/converge.js

```diff
@@ -1,22 +1,23 @@
+var _curry2 = require('./internal/_curry2');
 var _map = require('./internal/_map');
-var _slice = require('./internal/_slice');
 var curryN = require('./curryN');
 var pluck = require('./pluck');
 
 
 /**
- * Accepts at least three functions and returns a new function. When invoked, this new
- * function will invoke the first function, `after`, passing as its arguments the
- * results of invoking the subsequent functions with whatever arguments are passed to
- * the new function.
+ * Accepts a converging function and a list of branching functions and returns a new function.
+ * When invoked, this new function will invoke the first function, `after`, passing as its
+ * arguments the results of invoking the subsequent functions with whatever arguments are
+ * passed to the new function.
```

Comment:
"Subsequent functions" no longer makes sense. I suggest something along the lines of:

_When this new function is applied to some arguments, each branching function is applied to those same arguments. The results are then provided as arguments to the converging function to produce the return value._